py-polars/polars/testing/asserts.py

```diff
@@ -525,16 +534,14 @@ def raise_assert_detail(
     left: Any,
     right: Any,
     exc: AssertionError | None = None,
-) -> None:
+) -> NoReturn:
     """Raise a detailed assertion error."""
     __tracebackhide__ = True
 
     error_msg = textwrap.dedent(
         f"""\
-        {obj} are different.
-
-        {detail}
-        [left]:  {left}
+        {obj} are different ({detail})
+         [left]: {left}
```

Comment:
Not a fan of the right alignment here. Why did we have to change the format here? I thought it was fine as it was.

```suggestion
        {obj} are different ({detail})
        [left]:  {left}
```

Reply:
The right-align tends to line-up the diffs better so your eye jumps straight to the discontinuity, eg:
```
AssertionError: lazy schemas are not equal
 [left]: {'a': Int64, 'b': Float64, 'foo': Unknown}
[right]: {'a': Int64, 'b': Float64, 'foo': Int64}
```
vs
```
AssertionError: lazy schemas are not equal
[left]: {'a': Int64, 'b': Float64, 'foo': Unknown}
[right]: {'a': Int64, 'b': Float64, 'foo': Int64}
```

Reply:
There's an additional space after the `[left]: `, so the alignment of the data should be the same.

Reply:
Was there? I could also be going blind then :)

Reply:
Ahh, I can see your commit suggestion _does_ have an extra space! 
Ok, no strong feelings about that, let's commit it 🤣